src/app/globals.css

```
@@ -394,3 +394,66 @@ a.button.default {
+  max-height: min(600px, calc(100vh - 360px)); /* keeps actions visible */
+  overflow: auto;
+  padding: 12px;
+  border: 1px solid var(--win98-border-dark, #808080);
+  background: var(--win98-window-bg, #c0c0c0);
+}
+
+.cv-raw {
+  font-family: monospace;
+  font-size: 12px;
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.2.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)